package.json

```diff
@@ -9,11 +9,13 @@
     "build:holo-scale-test": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://scaletest-chaperone.holo.host/' vue-cli-service build",
     "build:self-hosted": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='self-hosted' vue-cli-service build",
     "build:holochain": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holochain' vue-cli-service build",
+    "build:holochain-zip": "yarn build:holochain && rm -f elemental-chat-ui.zip && (cd dist && rm service-worker.js && zip -r ../elemental-chat-ui.zip $(ls .))",
+    "build:holochain-webhapp": "yarn build:holochain-zip && hc web-app pack .",
     "test:unit-hc": "VUE_APP_CONTEXT='holochain' vue-cli-service test:unit --watch=true --config=./jest.hcunit.config.js",
     "test:unit-holo": "VUE_APP_CONTEXT='holo-host' vue-cli-service test:unit --watch=true --config=./jest.holounit.config.js",
-    "test-setup:happ": "nix-shell --command 'yarn fetch:dna && yarn build:happ' && exit",
-    "test-setup:holo": "yarn run clear:hc-holo && nix-shell --command 'yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc' && exit",
-    "test:integration-hc": "yarn run clear:hc && yarn run clear:lair && yarn build:holochain && nix-shell --command 'VUE_APP_CONTEXT='holochain' VUE_APP_WEB_LOGS='false' VUE_APP_TEST_PORT='8888' node scripts/test.js --config=./jest.hcintegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000' && exit",
```

Comment:
If you want to do this you need to see that you update the CI script

Reply:
ohhh good catch! that's why CI has been failing I bet